package.json

```diff
@@ -21,5 +21,17 @@
     "requirejs": "^2.1.17",
     "source-map": "^0.4.2",
     "through": "^2.3.7"
+    "del": "^0.1.3",
```

Comment:
Bad merge/rebase of package.json, there are some duplicate deps in here too. Will fix.